src/entrypoints/recorder.ts

```diff
@@ -386,7 +386,7 @@ function prepareRequest({
             timeOrigin,
             timestamp,
             method: method,
-            initiatorType: entry ? (entry.initiatorType as InitiatorType) : initiatorType,
```

Comment:
flips the choice here... if we're providing an initiator we should use it in preference to the detected one

they _should_ always match